wcc-contentful-app/app/views/components/_menu-item.html.erb

```diff
@@ -0,0 +1,26 @@
+<% return '' unless item
+   classes ||= []
+   classes.push('active') if item_active?(item)
+   classes.push('dropdown') if dropdown?(item) %>
```

Comment:
We should use the bootstrap 4 classes:

https://getbootstrap.com/docs/4.1/components/navs/#tabs-with-dropdowns

The resulting structure should mimic that.